### PR TITLE
Terms of service

### PR DESCRIPTION
**Pull request by martijn9612**

### CODE REVIEW

File: app/elements/routing.html [hunk 1]
```diff
@@ -27,6 +27,10 @@
     app.route = 'tickets-profile-intercept';
   });
 
+  page('/ToS', function () {
```
Comment (TimvdLippe) [inline app/elements/routing.html]: Do not use capitalization in routes.

File: app/elements/lancie-ticket-page/lancie-terms-of-service.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+-->
+
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../bower_components/iron-flex-layout/classes/iron-flex-layout.html">
+<link rel="import" href="../../bower_components/iron-ajax/iron-ajax.html">
+<link rel="import" href="../../bower_components/paper-card/paper-card.html">
+<link rel="import" href="../../bower_components/paper-material/paper-material.html">
+<link rel="import" href="../../bower_components/paper-input/paper-input.html">
+
+<dom-module id="lancie-terms-of-service">
```
Comment (martijn9612) [inline app/elements/lancie-ticket-page/lancie-terms-of-service.html]: This has to be added immediately due to legal concerns. Styling can be done later.

**Comment by skamoen** [comment]
So... Where are the actual terms?

**Comment by martijn9612** [comment]
Terms will be added tomorrow, terms are in plain text. We wanted to review but not merge this in advance to have this live as soon as possible.

**Comment by skamoen** [comment]
I would actually replace the checkout button action with opening the dialog. The dialog can have an accept and cancel button. The accept button would start the checkout process and proceed to Mollie. That way, you won't guide people away from your page and people have to click accept. 

Available right off the shelf here: https://elements.polymer-project.org/elements/paper-dialog?view=demo:demo/index.html&active=paper-dialog

File: app/elements/lancie-ticket-page/lancie-ticket-page.html [hunk 3]
```diff
@@ -48,6 +53,93 @@
         </lancie-ticket-selector>
       </lancie-section>
     </template>
+    <paper-dialog id="termsofservice" on-iron-overlay-opened="patchOverlay" entry-animation="scale-up-animation" exit-animation="fade-out-animation" with-backdrop>
+      <h2>Terms and conditions</h2>
+      <paper-dialog-scrollable>
+        <h3>0 Definitions</h3>
+        This document contains the general terms and conditions for “Area5LAN”, taking place from the 3rd of June until the 5th of June 2016, from now on referenced as “the event”. The event is established by “Student Association for Mathematics and Computer Science ‘Christiaan Huygens’” through the “LanCie Commission 2015-2016”, these two parties from now on referenced as “the organization”. Any other individual legally present at, or has the authority to be present at, the event and is not part of the organization is considered a participant, if not stated otherwise. Every individual at the event must follow these terms and conditions.
```
Comment (skamoen) [inline app/elements/lancie-ticket-page/lancie-ticket-page.html]: I'd only use the full "Area FiftyLAN" name. 

Student Association isn't the English for CH, That would be Mathematics and Computer Science Study Association 'Christiaan Huygens'

File: app/elements/lancie-ticket-page/lancie-ticket-page.html [hunk 4]
```diff
@@ -48,6 +53,93 @@
...
+        <h3>1 Validity</h3>
+        The organization reserves the right to modify these general terms and conditions until the start date of the event without any form of report of reason. The Dutch legislation is in effect at all times. Any violation of the terms and conditions will be punished by a fee of an appropriate amount unless specified otherwise.
+
+        <h3>2 Tickets</h3>
+        <h4>2.1 Purchasing tickets</h4>
+        Official tickets can only be purchased through the official website: https//areafiftylan.nl. Tickets retrieved through any other source will not be accepted. Purchasing a ticket automatically means the buyer accepts these terms and conditions. Any data given will not be shared with third parties. The tickets will be on sale until 1 (one) week before the actual event. After this, the ticket sale will be closed.
```
Comment (skamoen) [inline app/elements/lancie-ticket-page/lancie-ticket-page.html]: Two times 'official' in once sentence is weird. 

"Any data will not be shared" is odd. "No data will be shared with third parties" is better here.

File: app/elements/lancie-ticket-page/lancie-ticket-page.html [hunk 5]
```diff
@@ -48,6 +53,93 @@
...
+        <h4>2.2 Reselling your ticket</h4>
+        The reseller may charge the new buyer at maximum the original price; no profits must be made. The reseller is responsible for this exchange, including providing the organization with the right data.
+        <h4>2.3 Members of W.I.S.V. Christiaan Huygens</h4>
+        The event is, among others, hosted by the “Student Association for Mathematics and Computer Science Christiaan Huygens”. All members of this student association who wish to attend the event, can purchase a ticket for a lower price. Please note that your membership will be verified upon entering the event. If it appears you are not a member, but you still purchased a members-ticket, you will be asked to still pay the remaining costs of the ticket for a non-CH member. If you do not agree, you will be asked to leave the event.
```
Comment (skamoen) [inline app/elements/lancie-ticket-page/lancie-ticket-page.html]: Same here: Mathematics and Computer Science Study Association 'Christiaan Huygens'

` If it appears you are not a member, but you still purchased a members-ticket, you will be asked to still pay the remaining costs of the ticket for a non-CH member. If you do not agree, you will be asked to leave the event.` Those people will not be allowed in in the first place. I'd rephraze to this:

`Non-members in the possession of a members-ticket will have to pay the additional cost for a non-members ticket before entrance to the event is allowed.`

File: app/elements/lancie-ticket-page/lancie-ticket-page.html [hunk 6]
```diff
@@ -48,6 +53,93 @@
...
+        <h4>2.5 Tickets are sold out</h4>
+        If tickets appear to be sold out, one can be put on a waiting list. There will be no second round of the ticket sale, meaning the waiting list is the only way of getting a ticket. Once a ticket is available, you will be noted of the person trying to sell the ticket. The ticket resale will be done completely with the person trying to sell the ticket and the person trying to buy the ticket. The organization will do nothing more than providing the buyers contact info.
```
Comment (skamoen) [inline app/elements/lancie-ticket-page/lancie-ticket-page.html]: This is a weird way of stating what you mean. Not exactly sure how to phraze it better, will get back to this.

File: app/elements/lancie-ticket-page/lancie-ticket-page.html [hunk 7]
```diff
@@ -48,6 +53,93 @@
...
+        <h3>4 Liability</h3>
+        Only the participant is responsible for his personal and intellectual properties. Any instructions given by the organization must be followed. In case of unacceptable behavior, the participant may be given a warning or be asked to leave immediately. A warning is given once, thereafter follows removal from the event. If a participant is asked to leave the event, no refunds will be given.
+        <h3>5 Illegal activities</h3>
+        No weapons or other dangerous objects are allowed at the event. Hacking or spreading hazardous software, intentional or not, will result in removal of the event. If a participant is caught cheating in any way, he may not participate in any tournaments, including his current tournament. The participant will be removed from the event and will be banned from any future event. A participant will always coöperate with the organization when investigating sincere violations of the general terms and conditions. Otherwise, the participant will be removed from the event.
```
Comment (skamoen) [inline app/elements/lancie-ticket-page/lancie-ticket-page.html]: any future event**s**

File: app/elements/lancie-ticket-page/lancie-ticket-page.html [hunk 8]
```diff
@@ -48,6 +53,93 @@
...
+        <h5>6.3.2 Sales</h5>
+        No sales, which are not commissioned by the organization, are allowed at the event. If a participant is found selling products in any form, he will be removed from the event and a fine of  €500,- will be demanded as compensation. 
+        <h5>6.3.3 Surveys</h5>
+        Surveying at the event is allowed after the organization has given its permission. If unauthorized surveying is detected, the person responsible will be removed from the event. In case this person is a participant, he will be given a warning.
+        <h4>6.4 Alcohol/ narcotics</h4>
+        Alcohol consumation is only permitted above the age of 18. Drink with moderation or risk being removed from the event. Soft- and harddrugs, medication excluded, are not permitted at the event. Smoking is only permitted outside.
```
Comment (skamoen) [inline app/elements/lancie-ticket-page/lancie-ticket-page.html]: consumption

File: app/elements/lancie-ticket-page/lancie-ticket-page.html [hunk 9]
```diff
@@ -48,6 +53,93 @@
...
+        <h3>10 Tournaments</h3>
+        <h4>10.1 Registration</h4>
+        After purchasing a ticket, the participant can register himself for any tournament and change his registration up to one week before the starting of the event. If a tournament requires to enter as a team, one team member will make the reservation for all members and hold the rights for any changes to be made afterwards. This team member will automatically become the team captain in our system. The participant may only register for tournaments whom’s preliminary rounds do not begin at the same time. The official tournaments will always have priority over privately organised tournaments. This is done so a participant can participate in another tournament if it cannot continue in the other. In case he can participate in both, he must naturally chose either one and will be removed from the other.
```
Comment (skamoen) [inline app/elements/lancie-ticket-page/lancie-ticket-page.html]: You're actually closing registrations a week before? Why?

**Comment by skamoen** [comment]
Your lists in general don't work. Other then that, the dialog is looking good and works well. Don't forget to rebase.

**Comment by martijn9612** [comment]
This shoud be ready to merge. The terms and conditions are now up-to-date. Two small bullets are added to indicate pickup radius and parental approval.

File: app/elements/lancie-ticket-page/lancie-ticket-page.html [hunk 10]
```diff
@@ -9,9 +9,14 @@
 <link rel="import" href="lancie-ticket-list.html">
 <link rel="import" href="lancie-ticket-iron-block.html">
 <link rel="import" href="lancie-ticket-block.html">
+<link rel="import" href="../../bower_components/paper-styles/typography.html">
```
Comment (TimvdLippe) [inline app/elements/lancie-ticket-page/lancie-ticket-page.html]: In general we put `bower_component` imports before `lancie` imports.

File: app/elements/lancie-ticket-page/lancie-ticket-page.html [hunk 11]
```diff
@@ -43,7 +48,13 @@
                   </div>
                 </template>
                 <div class="card-actions">
-                  <paper-button raised on-tap="payOrder" disabled="{{hasNoTickets}}">Check out</paper-button>
+                    <ul style="list-style-type:circle">
```
Comment (TimvdLippe) [inline app/elements/lancie-ticket-page/lancie-ticket-page.html]: Remove this inline styling and put it in the .css

File: app/elements/lancie-ticket-page/lancie-ticket-page.html [hunk 12]
```diff
@@ -54,6 +65,93 @@
         </lancie-ticket-selector>
       </lancie-section>
     </template>
+    <paper-dialog id="termsofservice" on-iron-overlay-opened="patchOverlay" entry-animation="scale-up-animation" exit-animation="fade-out-animation" with-backdrop>
```
Comment (TimvdLippe) [inline app/elements/lancie-ticket-page/lancie-ticket-page.html]: This dialog got pretty long. To improve readibility (and make the patchOverlay hack more obvious) it is better to move this to a separate file.

**Comment by TimvdLippe** [comment]
Some small comments. For the other part LGTM

**Comment by TimvdLippe** [comment]
LGTM

**Pull request merged**